galvan/src/main/java/org/terracotta/testing/master/ServerProcess.java

```diff
@@ -217,16 +220,32 @@ private synchronized void reset(boolean running) {
   }
 
   private String[] buildCommand(boolean consistentStart) {
-    String startScript = getStartTcServerScript();
+    String startScript;
     String[] command;
-    if (consistentStart) {
-      command = new String[]{startScript, "-c", "-f", tcConfig.toString(), "-n", serverName, eyeCatcher};
+    if (configRepo != null) {
```

Comment:
Looks like creating specialized ServerProcess classes to handle servers with configRepo and tc-config would be cleaner here. Also, we can move these classes to tc-core and tc-platform to resolve other issues discussed in this PR.

Reply:
After the latest commit which makes Galvan agnostic of startup scripts and configs, this is gone